conf/application.conf

```diff
@@ -38,60 +38,47 @@ play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 play.i18n.langs = ["en", "cy"]
 
 microservice {
-    metrics {
-        graphite {
-            host    = localhost
-            port    = 2003
-            prefix  = play.${appName}.
-            enabled = false
-        }
+  metrics {
+    graphite {
+      host = localhost
+      port = 2003
+      prefix = play.${appName}.
+      enabled = false
+    }
+  }
+
+  services {
+    auth {
+      protocol = http
+      host = localhost
+      port = 8500
     }
 
-    services {
-      auth {
-        protocol = http
-        host     = localhost
-        port     = 8500
-      }
-
-      auth-login-stub {
-            protocol = http
```

Comment:
we don't define those urls here. Those are not (backend) "services" but just urls to other frontend apps.